templates/base.html

```diff
@@ -0,0 +1,21 @@
+<!DOCTYPE "html">
+<html lang="en">
+<head>
+  {% block title %}
+    <title>Taxi Service</title>
+  {% endblock %}
+  <meta charset="utf-8">
+  <meta name="viewport" content="width=device-width, initial_scale=1">
+  <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap@4.6.2/dist/css/bootstrap.min.css"
+        integrity="sha384-xOolHFLEh07PJGoPkLv1IbcEPTNtaed2xpHsD9ESMhqIYd0nLMwNLD69Npy4HI+N" crossorigin="anonymous">
+  {% load static %}
+  <link rel="stylesheet" type="text/css" href="{% static "css/styles.css" %}">
```

Comment:
You need to change the quotes in `<link rel="stylesheet" type="text/css" href="{% static 'css/styles.css' %}"> `because if you use "" inside them, you can only use ''